tests/Feature/Api/ApplicantSkillsControllerTest.php

```diff
@@ -15,7 +15,7 @@ public function testGetApplicantSkills(): void
     {
         $applicant = factory(Applicant::class)->create();
         $skills = Skill::inRandomOrder()->limit(5)->get();
-        $applicant->skills()->sync($skills->pluck('id'));
+        $applicant->skills()->sync($skills->sortBy('id')->pluck('id'));
```

Comment:
@yonikid15 could it be failing because there is no id to sort by in the array? what about `sort()` instead?

Reply:
There's no id in the collection of skill models?

Reply:
No, because it's no longer a collection after `pluck('id')`, only an array of the values. 

> [The pluck method retrieves all of the values for a given key.](https://laravel.com/docs/8.x/collections#method-pluck)